msmtools/analysis/dense/gpcca.py

```diff
@@ -230,7 +230,7 @@ def _do_schur(P, eta, m, z='LM', method='brandts', tol_krylov=1e-16):
         Q = _gram_schmidt_mod(Q, eta)
         # Transform the orthonormalized Schur vectors of P_bar back 
         # to orthonormalized Schur vectors X of P.
-        X = np.diag(1. / np.sqrt(eta)).dot(Q)
+        X = Q / np.sqrt(eta)[:, None]
```

Comment:
```suggestion
        X = np.true_divide(Q, np.sqrt(eta)[:, None])
```
Mb. we should use explicit true divison here (since in python 2 `/` will be interpreted as floor division).
Using `from __future__ import division` could be an additional precaution in general.